src/config.ts

```diff
@@ -12,23 +13,26 @@ const localnetConfig: ClusterConfig = {
 	...baseConfig,
 	name: SolanaCluster.LOCALNET,
 	RPCEndpoint: RPCEndpoint.LOCALNET,
+	stage: CivicStage.LOCALNET,
```

Comment:
this doesn't need to be part of the config. it's very specific for a particular component and can easily be derived from the cluster in the config already.